### DISCUSSION TITLE
Font inclusi nella libreria

### DISCUSSION THREAD

**Discussion by icolumbro-asf** [italia/design-angular-kit#273] [General]
Salve, ho iniziato a sviluppare un'app Angular utilizzando la libreria Design Angular Kit e seguendo attentamente le istruzioni presenti nella home per integrarla nell'applicazione. Sono ancora in una fase primordiale dello sviluppo però ho notato che il font utilizzato è diverso da quelli ufficiali di Bootstrap Italia e che vengono utilizzati anche nei file statici forniti da Developers Italia, come mai?

**Comment by AntoninoBonanno** [comment]
Dovresti aggiungere il tuo file css/scss come indicato nella [documentazione di bootstrap-italia](https://italia.github.io/bootstrap-italia/docs/come-iniziare/introduzione/#fonts)

oppure potresti usare nel tuo app.component.ts

```
import { loadFonts } from 'bootstrap-italia'

loadFonts('/bootstrap-italia/dist/fonts');
```

@astagi ha senso inserire nel kit il caricamento automatico dei font via javascript?

**Reply by icolumbro** [reply]
Grazie, sono riuscito a risolvere (funzionano entrambe le soluzioni suggerite). Adesso però avrei altri due problemi, uno piccolo, ossi che utilizzando il componente Steppers non riesco a far tradurre le etichette sui bottoni di navigazione ed uno un po' più grosso, non riesco ad utilizzare il componente it-upload-file-list perché, effettivamente, non si trova nel modulo che esporta i componenti, come mai?

**EDIT**: come non detto per il problema di it-upload-file-list, ero io che non ne avevo compreso bene l'utilizzo...

**Reply by AntoninoBonanno** [reply]
Per le traduzioni assicurati di aver configurato bene la libreria [https://github.com/italia/design-angular-kit#supporto-i18n-localizzazione](https://github.com/italia/design-angular-kit#supporto-i18n-localizzazione)

**Reply by icolumbro** [reply]
Approfitto della disponibilità: sto provando a capire come eseguire l'upload di un file guardando l'esempio della documentazione. Nell'esempio c'è la funzione httpClientExample() ma non capisco quando sia il momento di chiamarla... Quando clicco sul bottono di uoload e seleziono uno o più file, questi mi vengono aggiunti correttamente alla lista e li vedo elencati nel componente ma come faccio a far partire effettivamente l'upload del server?

**Reply by AntoninoBonanno** [reply]
Dovresti usarla nel `onUpdateFileList`, l'implementazione di `httpClientExample` (è un esempio per l'upload di un singolo file, ma puoi trasformarlo per caricare tanti file) dipende dal tuo server e dalle tue API.

Il concetto è che puoi modificare gli attributi di un elemento di `uploadedFileList` per mostrare l'avanzamento o eventuali errori

**Comment by AntoninoBonanno** [comment]
Con le ultime versioni, i font vengono caricati automaticamente da js. Puoi disattivare questa funzionalità dalla configurazione del modulo/provider
